bundles/binding/org.openhab.binding.dscalarm/META-INF/MANIFEST.MF

```diff
@@ -30,7 +30,9 @@ Import-Package: gnu.io,
  org.slf4j
 Service-Component: OSGI-INF/activebinding.xml, OSGI-INF/genericbindingprovider.xml
 Bundle-ClassPath: .
-Export-Package: org.openhab.binding.dscalarm
+Export-Package: org.openhab.binding.dscalarm,
+ org.openhab.binding.dscalarm.internal
+Bundle-ActivationPolicy: lazy
```

Comment:
I don't know why I am exporting .internal anyway. I don't think it is necessary.  I will change that.  Thanks @watou.